book/forms.rst

```diff
@@ -175,7 +175,7 @@ That's it! Just three lines are needed to render the complete form:
     Renders all the fields, which includes the field element itself, a label
     and any validation error messages for the field.
 
-``form_end()``
+``form_end(form)``
```

Comment:
hmm, sorry. It looks like it actually does something with the argument. I forgot it can do more than rendering the closing tag